lib/components/artist/articles/index.js

```diff
@@ -29,7 +29,7 @@ class Articles extends React.Component {
 
 const styles = StyleSheet.create({
   heading: {
-    marginBottom: 20,
+    marginBottom: 12,
```

Comment:
I checked and it's usually one of two cases:

1) The 20px padding is applied several times, which is easy - you remove it in one case
2) It's just off, but not explicitly states elsewhere - it's the content of the list items that's further indenting itself (because they don't start at 0, the have their own margins also for in between the items).